Code/Tools/AssetBundler/source/utils/applicationManager.cpp

```diff
@@ -198,7 +203,7 @@ namespace AssetBundler
     void ApplicationManager::SetSettingsRegistrySpecializations(AZ::SettingsRegistryInterface::Specializations& specializations)
     {
         AzToolsFramework::ToolsApplication::SetSettingsRegistrySpecializations(specializations);
-        specializations.Append("assetbundler");
+        specializations.Append(GetBuildTargetName());
```

Comment:
Is the intention here that BuildTargetName would be the _project name_ baked in?  I assume not, because with the installer version, we build this once, for all projects.

Reply:
The BuildTargetName does not have the project name baked in, this change is based on similar logic in [AssetProcessor](https://github.com/o3de/o3de/blob/development/Code/Tools/AssetProcessor/native/AssetProcessorBatchBuildTarget.cpp) and some other apps.